src/components/Authentication/Signup/Signup.jsx

```diff
@@ -0,0 +1,141 @@
+        <p className="text-7xl text-left mt-[99px] ml-[115px] font-bold">
+          Sign Up to Discover Startup Features
+        </p>
+        <div className="w-[470px] h-[324px] mt-[51px] ml-[115px]">
+          <div className="name text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.name && formik.touched.name ? "text-red-500" : ""
+              }`}
+            >
+              NAME
+            </label>
+            <input
+              type="text"
+              id="name"
+              placeholder="Your name"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.name && formik.touched.name ? "text-red-500" : ""
+              }`}
+              {...formik.getFieldProps("name")}
+            />
+            {formik.errors.name && formik.touched.name ? (
+              <div className="text-red-500">{formik.errors.name}</div>
+            ) : null}
+          </div>
+          <div className="email text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.email && formik.touched.email
+                  ? "text-red-500"
+                  : ""
+              }`}
+            >
+              Email:
+            </label>
+            <input
+              type="text"
+              id="email"
+              placeholder="Your email"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.email && formik.touched.email
+                  ? "text-red-500"
+                  : ""
+              }`}
+              {...formik.getFieldProps("email")}
+            />
+            {formik.errors.email && formik.touched.email ? (
+              <div className="text-red-500">{formik.errors.email}</div>
+            ) : null}
+          </div>
+          <div className="password text-lg text-left mt-6">
+            <label
+              className={`block font-medium ${
+                formik.errors.password && formik.touched.password
+                  ? "text-red-500"
+                  : ""
+              }`}
+            >
+              Password:
+            </label>
+            <input
+              type="password"
+              id="password"
+              placeholder="Your password"
+              className={`border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6 ${
+                formik.errors.password && formik.touched.password
+                  ? "text-red-500"
+                  : ""
+              }`}
+              {...formik.getFieldProps("password")}
+            />
+            {formik.errors.password && formik.touched.password ? (
+              <div className="text-red-500">{formik.errors.password}</div>
+            ) : null}
+          </div>
+          <form onSubmit={formik.handleSubmit}>
```

Comment:
The form inputs seem to be outside the form. This looks like a form with only a button and no inputs. What happens if I click submit ? Where do you console.log the values? Are there any values?